packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -582,6 +582,36 @@ OPTIONS added to them.
 See [#906](https://github.com/aws/aws-cdk/issues/906) for a list of CORS
 features which are not yet supported.
 
+### Endpoint Configuration
```

Comment:
Add links in this section to the APIGateway documentation which describe these options in detail. See the rest of the document for examples.